presto-docs/src/main/sphinx/release/release-0.262.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.262
+=============
+
+**Highlights**
+==============
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
+* Add support for simple TTL based scheduling based on worker availability estimates and query execution time estimates.
```

Comment:
we should not use abbreviations like TTL. Also, `simple` does not clarify anything here.

```
Add support for elastic workers by implementing time-to-live based node scheduling for queries with resource estimates. See :pr:`16409`
```